packages/record/src/recorder.ts

```diff
@@ -1,17 +1,38 @@
+/* eslint-disable jsdoc/no-types */
+/* eslint-disable jsdoc/no-defaults */
 import Data from "@lookit/data";
 import lookitS3 from "@lookit/data/dist/lookitS3";
 import autoBind from "auto-bind";
 import { JsPsych } from "jspsych";
 import { NoStopPromiseError, RecorderInitializeError } from "./error";
+// import MicCheckProcessor from './mic_check';  // TO DO: fix or remove this
+
+/**
+ * A valid CSS height/width value, which can be a number, a string containing a
+ * number with units, or 'auto'.
+ */
+type CSSWidthHeight =
+  | number
+  | `${number}${"px" | "cm" | "mm" | "em" | "%"}`
+  | "auto";
```

Comment:
Yeah. This might be overkill - the type could be 'string | number' or just 'string'. But since CSS values as parameters are tricky to check because of strings vs numbers and units, I thought it might be helpful for clarity to specify exactly what values should be used. Let me know if you think anything should be changed.